lib/ansible/utils/module_docs_fragments/spacewalk.py

```diff
@@ -0,0 +1,31 @@
+# Copyright: (c) 2018, Davide Blasi (@davegararh)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+class ModuleDocFragment(object):
+    # Parameters for Sapcewalk modules
+    DOCUMENTATION = '''
+options:
+    url:
+      description:
+      - The full URL to the Spacewalk API.
```

Comment:
Maybe add in an example, 

`The full URL to the Spacewalk API, such as C(https://spacewalk.example.net/rpc/api)`

The `C(...)` means Code formatting

Reply:
True, added